lib/chromic_pdf/pdf/browser/session_pool.ex

```diff
@@ -64,10 +70,15 @@ defmodule ChromicPDF.Browser.SessionPool do
      %{
        browser: browser,
        spawn_protocol: spawn_protocol(args),
-       max_session_uses: max_session_uses(args)
+       max_session_uses: max_session_uses(args),
+       timeout: timeout(args)
      }}
   end
 
+  defp timeout(args) do
+    get_in(args, [:session_pool, :timeout]) || @default_timeout
```

Comment:
Where is this imported? I wanted to make sure this is the Keyword.get_in not the Dict.get_in.

Reply:
this is in fact [`Kernel.get_in/2`](https://hexdocs.pm/elixir/Kernel.html#get_in/2) which uses the Access behaviour and works with both keyword lists and maps.